linters/README.md

```diff
@@ -5,3 +5,5 @@ Files to configure the linters we use for each language and technology
 
 * Ruby
   * [Rubocop](/linters/ruby/.rubocop.yml)
+* JavaScript
```

Comment:
No, only ES6. Although there is a version without React and another for legacy. See [here](https://github.com/airbnb/javascript). I could document how to change between them.